Cargo.toml

```diff
@@ -33,5 +33,6 @@ orml-tokens = { git = "https://github.com/open-web3-stack//open-runtime-module-l
 ### orml packages { version = "0.4.1-dev" }
 ### zenlink packages  { version = "*" }
 
-[patch."https://github.com/w3f/ring-vrf"]
-bandersnatch_vrfs = { git = "https://github.com/w3f//ring-vrf", version = "0.0.4", rev = "3ddc20" }
\ No newline at end of file
+# probable patch for nightly
```

Comment:
Is this not required anymore? In that case: can we remove it (instead of commenting it out)?

Reply:
My intention when I wrote the ticket was that whatever cargo command the pipeline runs (which was just plain `cargo test` when I wrote the ticket) it would be ran using the `runtime-benchmarks` feature so that the pipeline would catch any benchmarks related errors. So this is more like option 2 in my opinion.